packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -76,6 +80,11 @@ type SetForceReconfigureProjectByTestingType = {
   testingType?: TestingType
 }
 
+interface ReactComponentDescriptor {
+  description: string
```

Comment:
Is this unused? Seems that way.

Reply:
we're using this to type cast the result from `parseReactComponent` since the types are messed up
https://github.com/cypress-io/cypress/pull/24881/files#diff-d36f6626bf805d1bbbfba4786ca3a518f0083613a9814fd037c5592808949606R365

Reply:
Sorry, I should be more specific - I'm saying the `description` property is unused. We only use `displayName`. We can keep both if you think it's useful.

Reply:
Misunderstood - yeah, we don't need that property. Removed in https://github.com/cypress-io/cypress/pull/24881/commits/9449d66fbfadd25c70dad78bd243f6fde64b999d